### PR TITLE
Update READ_ONLY configuration for AWS.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

File: kitsune/settings.py [hunk 1]
```diff
@@ -503,13 +517,22 @@ def path(*parts):
     'commonware.middleware.XSSProtectionHeader',
     'commonware.middleware.RobotsTagHeader',
     # 'axes.middleware.FailedLoginMiddleware'
-)
+]
+if READ_ONLY:
```
Comment (pmac) [inline kitsune/settings.py]: You could also just include the middleware in the list and [mark it as unused at runtime](https://docs.djangoproject.com/en/1.8/topics/http/middleware/#marking-middleware-as-unused). Not a requirement, but an option if you're not happy with this. I'm happy either way, and this is perhaps a bit more explicit.

Comment (glogiotatidis) [reply kitsune/settings.py]: I like this, thanks!

**Comment by glogiotatidis** [comment]
@pmac updated with your suggestion

**Comment by pmac** [comment]
Looks great! 💯

**Comment by glogiotatidis** [comment]
many thanks!

**Pull request merged**